package.json

```diff
@@ -20,61 +20,61 @@
     "node": ">=12.22.6"
   },
   "dependencies": {
-    "address-rfc2821"       : "^2.0.1",
```

Comment:
Those formatting changes are not intended, perhaps the linter or my IDE did it, I'm not sure.